app/controllers/api/v1/reference/complex_cases_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Reference
+      class ComplexCasesController < ApiController
```

Comment:
I am not sure about the name `ComplexCases` ,
I had to read the `Importer` to have get an idea.
Are there also `SimpleCases`?

Maybe it's just me, when I hear about cases for me it's about court cases.

Reply:
Yeah I think I agree with @alexdesi, outside of the context of an "allocation" a "complex case" is a little confusing. If this is an entity specifically tied to only allocations we could use a similar pattern to `reference/prison_transfer_reasons` - it started out as just `reasons` but on their own outside of the context of a prison transfer they weren't useful.

I think Alessandro is also right with the term "case" being associated more with courts whereas here is feels like these items are like complex criteria for selecting prisoners. From what I gathered from @Andy-Millar there are used to force conversation (or not) with receiving prisons.

Reply:
I've renamed throughout to `AllocationComplexCase`, lets go with that name for now as it does align with the UX and prescribed business terminology. We'll have to just ignore the passing similarity with other nouns including `case` :)